Directory.Build.props

```diff
@@ -33,9 +33,10 @@
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
   </PropertyGroup>
   
-  <!-- Common NuGet package properties -->
+  <!-- Common version-related properties -->
   <PropertyGroup>
-    <PackageVersion Condition="'$(PackageVersion)'==''">0.0.1-alpha</PackageVersion>
+    <VersionPrefix Condition="'$(VersionPrefix)'==''">0.0.3</VersionPrefix>
```

Comment:
How does this behave if PackageVersion has been explicitly set?

Reply:
The explicitly set PackageVersion will override these. So your package will have one version and the assembly inside the package will have the version from VersionPrefix and VersionSuffix. Seems like the right behavior.

Reply:
I'm still trying to understand the versioning landscape here. Is it typical for a package version to be different than the assembly version?  
If PackageVersion isn't explicitly set, does it automatically get the value VersionPrefix-VersionSuffix?

Reply:
Also I understand that you probably used 0.0.3 because of your own internal usage and you need this to be monotonically increasing, but in the public repo we haven't released anything yet so starting at 0.0.3 isn't the right thing.

Reply:
Some NuGet packages have the same package version and assembly version, some don't. I seriously dislike not having them be in sync because given an assembly and knowing its version, I don't have to guess what version of the NuGet package it came from.

Yes if the PackageVersion is not explicitly set it gets the value VersionPrefix-VersionSuffix automatically.

Also, yes, I need the version to be increasing. Since the public repo isn't publishing a NuGet package yet does it even matter what the version is? I am publishing my own NuGet package for Core WCF since there isn't an official one yet. 

If you don't want my version, what would you like the version to be set to in the public repo?

Reply:
I suppose it can stay at 0.0.3 for now as we're not publishing anything but be aware that when I get the versioning stuff all hooked up with GitVersion, this will change.

Reply:
Sure, I can't wait for the day that I don't have to publish my own package any longer.

Reply:
I personally prefer the approach taken by EF where assembly version is kept at major version of the nuget package. But you should keep the informal version and file version in sync with the nuget and commit used (via sourcelink)

That way you keep binding redirects at a minimum and can always easily see what nuget version (and commit) corresponds to a dll